paper.md

```diff
@@ -309,9 +298,9 @@ The training step was run using MPI on 2 compute nodes having a GPU each and the
 
 As we can see from @Fig:casestudies, Popper allowed us to run the workflow in a Slurm cluster with MPI and hence utilize the processing power of multiple GPUs and drastically reduce the training duration.
 
-### Setting up CI for our project
+### Workflow execution on CI
 
-We pushed our MNIST project to GitHub and activated the repository in Travis to set up continuous integration on our project.
+We pushed our MNIST project to GitHub and activated the repository in Travis to run our workflows on CI.
```

Comment:
we can say that we generated a configuration for travis using the workflow exporter explained earlier